README.md

```diff
@@ -331,7 +331,7 @@ fastify.decodeSecureSession(request.cookies['session'], undefined, 'mySecondSess
 
 ## Add TypeScript types
 
-The session data is typed as `{ [key: string]: any }`. This can be extended with [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) to get improved type support.
+The session data is defined as interface `SessionData`. To extended `SessionData` with [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) for improved type support.
```

Comment:
```suggestion
The session data is defined as an interface called `SessionData`. It can be extended with [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) for improved type support.
```